### PR TITLE
Delete FixedPackages.props

### PR DESCRIPTION
**Pull request by jasonmalinowski**
None of the properties in this file were consumed anywhere in our build, so clearly this is all a lie.

If anybody can tell me that some of these were supposed to be consumed, I'm happy to update the code appropriately, but until then this is only causing confusion for people.

### CODE REVIEW

**Comment by jasonmalinowski** [comment]
Ah, it's being used outside Roslyn.sln's build, so this will require a bit more precision.

File: build/Targets/FixedPackages.props [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-    <MicrosoftVisualStudioTextLogicFixedVersion>14.0.23205</MicrosoftVisualStudioTextLogicFixedVersion>
-    <MicrosoftVisualStudioTextUIFixedVersion>14.0.23205</MicrosoftVisualStudioTextUIFixedVersion>
-    <MicrosoftVisualStudioTextUIWpfFixedVersion>14.0.23205</MicrosoftVisualStudioTextUIWpfFixedVersion>
-    <MicrosoftVisualStudioUtilitiesFixedVersion>14.0.23205</MicrosoftVisualStudioUtilitiesFixedVersion>
-    <MicrosoftVSSDKBuildToolsFixedVersion>14.3.25420</MicrosoftVSSDKBuildToolsFixedVersion>
-    <MicrosoftVisualStudioComponentModelHostFixedVersion>14.0.25424</MicrosoftVisualStudioComponentModelHostFixedVersion>
-    <NETStandardLibraryFixedVersion>2.0.0-beta-25123-01</NETStandardLibraryFixedVersion>
-    <NewtonsoftJsonFixedVersion>9.0.1</NewtonsoftJsonFixedVersion>
-    <SystemCollectionsImmutableFixedVersion>1.1.36</SystemCollectionsImmutableFixedVersion>
-    <SystemReflectionMetadataFixedVersion>1.0.21</SystemReflectionMetadataFixedVersion>
```
Comment (jaredpar) [inline build/Targets/FixedPackages.props]: You're undoing all the terrible work I did to undo other terrible things.

**Comment by jaredpar** [comment]
What other issues did you hit? I was going to point out you missed a buildboss part

https://github.com/dotnet/roslyn/blob/master/src/Tools/BuildBoss/ProjectCheckerUtil.cs#L223

But that shouldn't have blocked anything.

**Comment by jasonmalinowski** [comment]
@jaredpar Looks like the MSBuild task project does reference just the msbuild portion out of that. I'm at a loss how my git grep missed this.

```
    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildFixedVersion)" />
    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreFixedVersion)" />
```

Does the "Fixed" thing mean anything anymore here? I assume you want to keep MSBuild 14 as your lower bound here, wondering if we should just name the variable something more like "MSBuildForBuildTasksVersion" so it's clear what's fixing it vs. "this is some fixed version by...something"

**Comment by jasonmalinowski** [comment]
I'm happy of course to put the files back and just delete everything else. :smile:

**Pull request closed without merging**